src/constants.js

```diff
@@ -1,7 +1,3 @@
-export const DISPLAY_FORMAT = 'L';
-export const ISO_FORMAT = 'YYYY-MM-DD';
-export const ISO_MONTH_FORMAT = 'YYYY-MM'; // TODO delete this line of dead code on next breaking change
```

Comment:
removing these exports is a breaking change

Reply:
Yeah I figured it would be